content/release-10-18-0/edge-10-18-0-bundle/10_18_0.md

```diff
@@ -0,0 +1,58 @@
+---
+weight: 60
+title: Release 10.18.0
+layout: redirect
+---
+
+The release notes for **Cumulocity IoT Edge release 10.18.0** contain important information about all the changes, including new features, enhancements, and fixes that have been made to both the **Cumulocity IoT Edge Appliance VM** and the **Cumulocity IoT Edge on Kubernetes**. If you want to know more, you can check out the [What´s New](/release-10-18-0/whatsnew-10-18-0/) section and watch our video [What's new in Cumulocity IoT 10.18](https://www.youtube.com/watch?v=pTouaQbc6SI&list=PLexm-BA6Wrr8AYnu97vpIR7uicD2wsyms).
+
+Additionally, the release notes contain some [announcements](/release-10-18-0/edge-10-18-0/#announcements) regarding planned and implemented deprecations that might require some action on your part at some point.
+
+
+You can find the corresponding documentation for **Cumulocity IoT Edge Appliance VM** at https://cumulocity.com/guides/10.18.0/edge/edge-introduction/ and for **Cumulocity IoT Edge on Kubernetes** at https://cumulocity.com/guides/10.18.0/edge-k8s/k8-edge-introduction/
+
+### Cumulocity IoT Edge Appliance VM
+
+#### Rocky Linux 8
+As CentOS 7 will reach its end-of-life on June 30th, 2024, we are transitioning our support to an alternative Operating System for Edge Appliance users. Starting with this release, the Edge Appliance VM is equipped with [Rocky Linux 8](https://rockylinux.org/). Since the Operating System is changed, the upgrade from earlier version of Edge entails a side-by-side installation accompanied by data migration. For information on upgrading Edge version 10.18, see [Migrating Edge 10.17 to 10.18](https://cumulocity.com/guides/10.18.0/edge/update-bundle/migration-1017-to-1018/).
```

Comment:
```suggestion
As CentOS 7 will reach its end-of-life on June 30th, 2024, we are transitioning our support to an alternative Operating System for Edge Appliance users. Starting with this release, the Edge Appliance VM is equipped with [Rocky Linux 8](https://rockylinux.org/). Since the Operating System is changed, the upgrade from earlier versions of Edge entails a side-by-side installation accompanied by data migration. For information on upgrading Edge version 10.18, see [Migrating Edge 10.17 to 10.18](https://cumulocity.com/guides/10.18.0/edge/update-bundle/migration-1017-to-1018/).
```